src/components/DarkModeToggle/DarkModeToggle.component.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Container, ToggleOffIcon } from './DarkModeToggle.styled';
+
+const DarkModeToggle = () => {
+  // Later update the icon depends on the darkmode on/off state.
+  const getIcon = () => <ToggleOffIcon />;
+
+  return (
+    <Container>
+      {getIcon()}
```

Comment:
Good! You can store the state and code something like `{isDarkOn ? <ToggleOffIcon /> : <ToggleOnIcon />}`, we'll learn how to store the state when learning about Hooks & State Management